app/components/GenreCard/tests/index.test.js

```diff
@@ -2,13 +2,36 @@ import React from 'react';
 import { render } from 'react-testing-library';
 // import $ from 'jquery';
 import GenreCard from '../index';
+import { DefaultState, ActiveState } from '../GenreCardButtonStyle';
 
 describe('<GenreCard />', () => {
```

Comment:
test: `onClick` is called when the component is clicked. Google `jest.fn()` for the callback mock.